File: apps/monolith/src/main/java/com/example/iter/admin/payment/dto/AdminPaymentRentalResponse.java
package com.example.iter.admin.payment.dto;

import com.example.iter.reservation.api.RentalStatus;

import java.math.BigDecimal;
import java.time.LocalDate;

public record AdminPaymentRentalResponse(
        Long equipmentId,
        String equipmentName,
        String category,
        BigDecimal dailyPrice,
        LocalDate startDate,
        LocalDate endDate,
        Integer rentalDays,
        BigDecimal totalPrice,
        RentalStatus rentalStatus
) {
}

File: apps/monolith/src/main/kotlin/iter/admin/payment/dto/AdminPaymentRentalResponse.kt
package iter.admin.payment.dto

import iter.reservation.api.RentalStatus
import java.math.BigDecimal
import java.time.LocalDate
import kotlin.jvm.JvmRecord

@JvmRecord
data class AdminPaymentRentalResponse(
    val equipmentId: Long?,
    val equipmentName: String?,
    val category: String?,
    val dailyPrice: BigDecimal?,
    val startDate: LocalDate?,
    val endDate: LocalDate?,
    val rentalDays: Int?,
    val totalPrice: BigDecimal?,
    val rentalStatus: RentalStatus?
)
